frontend/app/src/pages/main/v1/workflows/$workflow/components/trigger-workflow-form.tsx

```diff
@@ -64,6 +64,29 @@ export function TriggerWorkflowForm({
     defaultWorkflow?.metadata.id,
   );
 
+  const [workflowSearch, setWorkflowSearch] = useState<string>('');
+  const [debouncedWorkflowSearch, setDebouncedWorkflowSearch] =
+    useState<string>('');
```

Comment:
minor TS nit: we don't need the `<string>` here since the compiler can infer that from the default empty string